drivers/counter/Kconfig.native_posix

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config COUNTER_NATIVE_POSIX
+    bool
+    default y
+    depends on BOARD_NATIVE_POSIX
+
+config COUNTER_FREQUENCY
```

Comment:
Better name it something like COUNTER_NATIVE_POSIX_FREQUENCY (or so, otherwise the option name is too generic)